frontend/packages/console-app/console-extensions.json

```diff
@@ -502,14 +506,16 @@
     }
   },
   {
-    "type": "console.navigation/resource-ns",
+    "type": "console.navigation/href",
     "properties": {
       "perspective": "admin",
       "section": "home",
       "id": "events",
       "insertAfter": "explore",
       "name": "%console-app~Events%",
-      "resource": "events"
+      "href": "/events",
+      "prefixNamespaced": true,
+      "startsWith": ["events"]
```

Comment:
Is this needed? Shouldn't by the nature of having a `href` cover this?

Reply:
This is probably not strictly necessary. It might have just been a left-over prop from the original admin nav component.